hip_data_tools/aws/athena.py

```diff
@@ -424,7 +424,9 @@ def extract_tables_from_explaination(self):
         for row in explaination_rows:
             for data in row['Data']:
                 refs = self.parse_table_entries(data['VarCharValue'])
-                self.table_schema_entries.extend(refs)
+                for table_entry in refs:
```

Comment:
These 3 nested for loops increases the cyclomatic complexity. We can break down one for loop into a private method.